### PR TITLE
chore(Icons): update icons

### PR DESCRIPTION
**Pull request by marcoskolodny**
The 90% of the changes are minor differences caused by regenerating all the icons

New icons component:
- video-camera-disabled
- microphone-aura
- microphone-disabled

New icons variant included for other brands:
- mute
- thumb-down

### CODE REVIEW

**Comment by yceballost** [comment]
You can check the new icons in this commit https://github.com/Telefonica/mistica-web/pull/1278/commits/846da174e9c3c76aeeb038031efa767837e837c4

File: src/generated/mistica-icons/icon-mute-filled.tsx
Comment (yceballost) [inline src/generated/mistica-icons/icon-mute-filled.tsx:1]: New for O2

File: src/generated/mistica-icons/icon-mute-light.tsx
Comment (yceballost) [inline src/generated/mistica-icons/icon-mute-light.tsx:1]: New for O2

File: src/generated/mistica-icons/icon-mute-regular.tsx
Comment (yceballost) [inline src/generated/mistica-icons/icon-mute-regular.tsx:1]: New for Telefonica set

File: src/generated/mistica-icons/icon-thumb-down-filled.tsx
Comment (yceballost) [inline src/generated/mistica-icons/icon-thumb-down-filled.tsx:1]: New for O2

File: src/generated/mistica-icons/icon-thumb-down-light.tsx
Comment (yceballost) [inline src/generated/mistica-icons/icon-thumb-down-light.tsx:1]: New for O2

File: src/generated/mistica-icons/icon-thumb-down-regular.tsx
Comment (yceballost) [inline src/generated/mistica-icons/icon-thumb-down-regular.tsx:1]: New for O2

**Review by yceballost — APPROVED** [review]

**Review by atabel — APPROVED** [review]

**Pull request merged**